### PR TITLE
Lesson4

### PR DESCRIPTION
**Pull request by LudmilaZhuleha**
Function sum()() is done.
Coloring paragraphs is done. But the first click miss the color change...all other clicks are ok. Please have a look. There must be somrthing i looked through.
Jest tests have not been done yet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,31 +3,35 @@
...
-    "lint": "eslint --debug src/",
-    "lint:write": "eslint --debug src/ --fix",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/LudmilaZhuleha/if-js.git"
   },
+  "scripts": {
+    "prettier": "prettier --write src/**/*.js",
```
Comment (Marinarium) [inline package.json:11]: "prettier": "prettier --write {src,tests}/**/*.js",

File: index.html [hunk 2]
```diff
@@ -8,7 +8,10 @@
     <title>JS</title>
 </head>
 <body>
-
-<script src="src/main.js"></script>
+<p id="text1">Text1</p>
+<p id="text2">Text2</p>
+<p id="text3">Text3</p>
+<script type="module" src="src/main.js"></script>
+<script type="module" src="tests/changeColor.test.js"></script>
```
Comment (Marinarium) [inline index.html]: delete

**Review by Marinarium — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (LudmilaZhuleha) [reply package.json:11]: Done

File: index.html (hunk 2 above)
Comment (LudmilaZhuleha) [reply index.html]: Done.

**Pull request merged**